scripts/docker-build.sh

```diff
@@ -9,6 +9,7 @@ DOCKER_BUILD_CMD=(docker buildx build --load -f Dockerfile
   --build-arg PIP_VERSION="$PIP_VERSION"
   --build-arg BUILDKIT_INLINE_CACHE=1
   --progress plain
+  --platform linux/amd64
```

Comment:
For building the image locally since it doesn't work otherwise on Mac. The `docker-build` workflow should still build it for both ARM and AMD architectures.